lib/graphql/transaction.ts

```diff
@@ -55,9 +70,22 @@ export class Transaction extends IterableModel<TransactionEntry> {
   async fetch(args?: FetchOptions): Promise<ResultPage<TransactionEntry>> {
```

Comment:
The changes to this method are unrelated to my feature, I use prettier to format for me and it did these formatting changes, I can remove them if necessary